b2b-sample/guardio-insurance-sample-application/components/sections/sections/settingsSection/roleManagementSection/otherComponents/roleItemDetailsSection/permission.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useCallback, useEffect, useState } from "react";
+import { Field, Form } from "react-final-form";
+import { Button, ButtonToolbar, CheckTree, Loader, useToaster } from "rsuite";
+import FormSuite from "rsuite/Form";
+import styles from "../../../../../../../styles/Settings.module.css";
+import decodePatchRole from "../../../../../../../util/apiDecode/settings/role/decodePatchRole";
+import { PatchMethod } from "../../../../../../../util/util/common/common";
+import { LOADING_DISPLAY_BLOCK, LOADING_DISPLAY_NONE } from "../../../../../../../util/util/frontendUtil/frontendUtil";
+import { errorTypeDialog, successTypeDialog } from "../../../../../../common/dialog";
+import orgRolesData from "../../data/orgRolesData.json";
+
+/**
+ * 
+ * @param prop - `fetchData` - function , `session`, `roleDetails` - Object
+ * 
+ * @returns The permission section of role details
+ */
+export default function Permission(prop) {
+
+    const { fetchData, session, roleDetails } = prop;
+
+    const [ loadingDisplay, setLoadingDisplay ] = useState(LOADING_DISPLAY_NONE);
+    const [ selectedPermissions, setSelectedPermissions ] = useState([]);
+
+    const toaster = useToaster();
+
+    const setInitialPermissions = useCallback(async () => {
+        setSelectedPermissions(roleDetails.permissions);
+    }, [ roleDetails ]);
+
+    useEffect(() => {
+        setInitialPermissions();
+    }, [ setInitialPermissions ]);
+
+    const onDataSubmit = (response, form) => {
+        if (response) {
```

Comment:
make the code consistent by leaving a new line after declarion.

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/05bcb70be881109f31c421be63d5d5d884094b42